meta/bounty-definition-template.md

```diff
@@ -0,0 +1,76 @@
+* Relevant tags? (See existing bounties and reuse where possible, new tags are recommended if sufficiently distinct)
+* Min reward (USD)?
+* Max reward (USD)?
+* Any linked definitions? (e.g. if a single bounty proposal becomes multiple definitions.)
+* Any dependencies? 
+
+## Summary 
+Short 1-3 sentences describing the bounty.
+
+## Dependencies
+Is there anything that must happen (outside of a bounty hunter's control) before it is possible to complete this bounty? This may be other bounties that must be completed first, an upcoming event or change or a regular occurance that triggers a valid bounty. This section is optional. May be later removed from the definition if the dependency becomes permanently met. 
+
+## Required Milestones
+What _must_ be completed for a bounty hunter to claim some amount of bounty. Described per milestone.
+
+### Milestone A - <Name of Milestone>
+**Payout: ** <payout amount>
+Clear bulleted list or subheadings covering the items that must be completed and/or adhered to for this milestone to be valid.
+
+### Milestone B - <Name of Milestone>
+**Payout: ** <payout amount>
+Clear bulleted list or subheadings covering the items that must be completed and/or adhered to for this milestone to be valid.
+
+### Milestone C - <Name of Milestone>...
+
+## Optional Milestones
+What tasks _may_ be completed for a bounty hunter to earn extra bounty rewards. Described per milestone. This section is optional.
+
+Optional milestones may be less strictly defined than required milestones. You may aggregate multiple minor considerations that would contribute to a payout. 
+
+### Milestone D - <Name of Milestone>
+**Maximum Payout: ** <maximum payout amount>
+Clear bulleted list of the items that would contribute to payout for this milestone.
+
+### Milestone E - <Name of Milestone>...
+
+
+## Further Notes
+Anything you think that would be beneficial for a bounty hunter to know when working on this bounty. Maybe be divided into subsections as needed.
+
+## Resources
+Links to repositories, web pages, forum discussions, etc. Anything that the bounty hunter may be able to use to do a better job on the bounty work. 
+
+## Contacts
+Individuals that have agreed to act as contacts for this bounty. Include usernames + contact details for any platform on which the contact is willing to respond to requests. Any contacts are expected to fully understand the bounty definition. This section is optional. 
+
+Contacts:
+* MAY be eligible for incentives.
+* SHOULD NOT assist the bounty hunter directly with the bounty work.
+* SHOULD assist bounty hunters via feedback, direction and oversight upon request.
+
+## Verification
+Who is expected to verify that the work delivered meets the relevant milestones? This person or group must have agreed to do this in advance of this definition being published. This person or group should have any relevant skillsets needed to properly verify the bounty work.
+
```

Comment:
Will applicants get stuck here? I don't think most proposers have any idea who will or can verify their work (aside from the regulars like ramana). Should we change the language to request suggested parties for verification?